cpu/stm32f1/ldscripts/.gitignore

```diff
@@ -0,0 +1 @@
+/stm32f103re-slots.ld
```

Comment:
Yes, I don't want to include this .ld file into the RIOT source, since it can change if the slot sizes change.